app/views/layouts/application.html.erb

```diff
@@ -24,6 +24,9 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+<!--[if ( !IE ) | ( gte IE 9 ) ]><!-->
+<%# render 'dough/shared/social_sharing_icon_sprite' %>
```

Comment:
I *think* it is just so as to not load the SVG icons into a browser that doesn't understand them. (But yes, it is the same as frontend).